packages/netlify/package.json

```diff
@@ -19,11 +19,8 @@
   "homepage": "https://docs.astro.build/en/guides/integrations-guide/netlify/",
   "exports": {
     ".": "./dist/index.js",
-    "./functions": "./dist/integration-functions.js",
-    "./static": "./dist/integration-static.js",
-    "./netlify-functions.js": "./dist/netlify-functions.js",
-    "./edge-functions": "./dist/integration-edge-functions.js",
-    "./netlify-edge-functions.js": "./dist/netlify-edge-functions.js",
```

Comment:
At Astro we don't delete stuff from one major to another, but we first deprecate things in a major, and delete it in the next major.

This means that we **can't** remove these entry points. 

Two other important things:
- I would argue that `/static` and `/serverless` are more descriptive than `/ssr-function`
- Why did we remove `/static`? Does this mean that we removed static support in the netlify adapter?

Reply:
> I would argue that /static and /serverless are more descriptive than /ssr-function

`ssr-function` is not meant for users, but only for passing something to the Astro Adapter: https://github.com/Skn0tt/adapters/blob/bd7dbba6c23e815f23bd73a69b43bf0504d81e00/packages/netlify/src/index.ts#L194
In that usecase, i'd say it's pretty descriptive.

> Why did we remove /static? Does this mean that we removed static support in the netlify adapter?

The adapter still supports static deployments, it's all inside the one main entrypoint. For static sites, we skip generating lambdas, but still generate redirects:

https://github.com/Skn0tt/adapters/blob/bd7dbba6c23e815f23bd73a69b43bf0504d81e00/packages/netlify/src/index.ts#L218-L229

> At Astro we don't delete stuff from one major to another, but we first deprecate things in a major, and delete it in the next major.

Thanks for pointing that out! In the current adapter, there's exports `/functions`, `/edge-functions` and `/static`. I can point `/static` and `/functions` towards the main entrypoint, and add a deprecation warning.
For `/edge-functions`, we'd probably have to add support for deploying to edge functions - but that's good to have, anyways.

Reply:
I've re-added `/static` and `/functions` in https://github.com/withastro/adapters/pull/84/commits/580820c78c4127339b833f1464936d26e0dbfc2a.

For `/edge-functions`, I did update the adapter to support it in https://github.com/withastro/adapters/pull/84/commits/3a03eb80f4513099278d2be9cbe6a7e234b08e18. Then realised that v3 had an `exports` field for it, but the implementation was removed in https://github.com/withastro/adapters/commit/19ea8b7a1668939c275e94b94ae159a3f8617c55. So I think it's fine to leave that out here, as well.